### DISCUSSION TITLE
Unable to transfer files even though my Android phone and Windows device are connected to same network. but when the devices are connected through my mobile hotspot the transfer works without any issue

### DISCUSSION THREAD

**Discussion by amruthmonnappa** [localsend/localsend#1428] [Discussion (English, International)]
I have been facing an issue with transferring files between my Android phone and Windows device. Despite both devices being connected to the same Wi-Fi network, the file transfer simply doesn't work. This problem has been quite frustrating, especially because the connection is stable and both devices are easily recognizable on the network.

Interestingly, when I connect both devices using my mobile hotspot, the file transfer works seamlessly without any issues.

**Comment by JannetGen** [comment]
It can let both devices access the internet normally while blocking them from talking directly to each other. Since the hotspot works, I’d check the router’s guest network/client isolation settings first. As a quick workaround, you could also try [FileMonkey](https://filemonkey.org/) for browser-to-browser Wi-Fi transfers.
